docs/readthedocs/source/doc/LLM/DockerGuides/docker_run_pytorch_inference_in_vscode.md

```diff
@@ -0,0 +1,135 @@
+# Run/Develop PyTorch in VSCode with Docker on Intel GPU
+
+An IPEX-LLM container is a pre-configured environment that includes all necessary dependencies for running LLMs on Intel GPUs. 
+
+This guide provides steps to run/develop PyTorch examples in VSCode with Docker on Intel GPUs.
+
+```eval_rst
+.. note::
+
+   This guide assumes you have already installed VSCode in your environment. 
+   
+   To run/develop on Windows, install VSCode and then follow the steps below. 
+   
+   To run/develop on Linux, you might open VSCode first and SSH to a remote Linux machine, then proceed with the following steps.
+
+```
+
+
+## Install Docker
+
+Follow the [Docker installation Guide](./docker_windows_gpu.html#install-docker) to install docker on either Linux or Windows.
+
+## Install Extensions for VSCcode
+
+#### Install Dev Containers Extension
+For both Linux/Windows, you will need to Install Dev Containers extension.
+
+Open the Extensions view in VSCode (you can use the shortcut `Ctrl+Shift+X`), then search for and install the `Dev Containers` extension.
+
+
+![install_dev_container-ezgif com-video-to-gif-converter (1)](https://github.com/liu-shaojun/ipex-llm/assets/61072813/21fbbd60-8d3e-4b6d-949b-62f43a102148)
+
+
+
+#### Install WSL Extension for Windows
+
+For Windows, you will need to install wsl extension to to the WSL environment. Open the Extensions view in VSCode (you can use the shortcut `Ctrl+Shift+X`), then search for and install the `WSL` extension.
+
+Press F1 to bring up the Command Palette and type in `WSL: Connect to WSL Using Distro` and select it and then select a specific WSL distro `Ubuntu`
+
+![installwslextention-ezgif com-video-to-gif-converter (1)](https://github.com/liu-shaojun/ipex-llm/assets/61072813/f53b9d17-1719-442c-b08f-d0c369476dc0)
+
+
+## Launch Container
+
+Open the Terminal in VSCode (you can use the shortcut `Ctrl+Shift+``), then pull ipex-llm-xpu Docker Image:
```

Comment:
what is the shortcut
it shows Ctrl+Shift+&#96;  in the doc

Reply:
It should be `` Ctrl+Shift+` ``, it is to open the terminal in VSCode, already updated.